apps/ios/Shared/Model/SimpleXAPI.swift

```diff
@@ -567,9 +567,10 @@ func apiNewGroup(_ p: GroupProfile) throws -> GroupInfo {
     throw r
 }
 
-func addMember(groupId: Int64, contactId: Int64, memberRole: GroupMemberRole) async {
+func addMember(groupInfo: GroupInfo, contactId: Int64, memberRole: GroupMemberRole) async {
```

Comment:
that's not a good idea in general to handle errors here, as you cannot show any alerts here - why can't it be handled at call site?